src/plugins/DevTools/components/DevToolsComponent.ts

```diff
@@ -51,6 +49,20 @@ export class DevToolsComponent extends OwlUIComponent<{}, DevToolsState> {
             actionsComponent.addAction(action);
         }
     }
+    /**
+     * On render the `vDocument`, rerender and select the range nodes.
+     */
+    onRender(): void {
+        this.render();
+        const inspectorComponent = this.refs.InspectorComponent as InspectorComponent;
+        if (inspectorComponent) {
+            const rangeNodes = this.env.editor.vDocument.rangeNodes;
+            inspectorComponent._selectNode(rangeNodes.start);
+            // In order for owl-framework to unfold the parents of start *and*
+            // of end, we need to select both, with a tick in between.
+            setTimeout(() => inspectorComponent._selectNode(rangeNodes.end), 0);
```

Comment:
In the function that computes whether a node should be folded or not, why not check if it contains the range instead ?